### PR TITLE
Added support for multiple messages files

### PR DESCRIPTION
**Pull request by justinpurcell1**
#11 

Note that the Object.assign means that file order matters if the same key exists in multiple files.

I can add unit tests as needed, but I figured I would at least get eyes on this.

### CODE REVIEW

File: util.js [hunk 1]
```diff
@@ -52,13 +52,19 @@ module.exports = {
...
   },
 
-  readMessages: (messagesFilepath, locale) => {
-    messagesFilepath = messagesFilepath.replace("[locale]", locale);
-    if (!fs.existsSync(messagesFilepath) || !fs.statSync(messagesFilepath).isFile()) {
-      console.warn("Unable to find messages file: `" + messagesFilepath + "`");
-      return null;
+  readMessages: (messagesFilepaths, locale) => {
+    let messages = {};
+    const filepaths = [].concat(messagesFilepaths);
```
Comment (rxaviers) [inline util.js]: Great solution to make it always an array.

**Comment by rxaviers** [comment]
Please, can you add a test?

**Comment by justinpurcell1** [comment]
@rxaviers I have added unit tests, however when "es" is included in the supported locales , there is a module parse failure. I'm having some trouble locating the root of this problem.

File: test/fixtures/more-translations/es.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "en": {
```
Comment (rxaviers) [inline test/fixtures/more-translations/es.json]: You need to update this to `es`.

**Comment by justinpurcell1** [comment]
Fixed

**Comment by rxaviers** [comment]
Thanks

**Pull request merged**